Scripts/new_feature.sh

```
@@ -0,0 +1,116 @@
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
+        XCTAssertTrue(true)
+    }
+
+}
+EOF
+
+# Add to app dependencies
+LOWER_CAMEL_CASE_FEATURE_NAME="$(tr '[:upper:]' '[:lower:]' <<< "${FEATURE_NAME:0:1}")${FEATURE_NAME:1}"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, here-strings are undefined.

(SC3011)